internal/system/swap.go

```diff
@@ -56,14 +56,22 @@ func (s *swapAspect) Setup() error {
 // If partition type swap exists, user needs to manually remove the partition swap before
 // running nodeadm init.
 func partitionSwapExists() (bool, error) {
-	cmd := "swapon -s | awk '$2==\"partition\" {print}'"
-	out, err := exec.Command("bash", "-c", cmd).CombinedOutput()
+	f, err := os.Open("/proc/swaps")
 	if err != nil {
```

Comment:
should we check here if the error is file not found?
IMO there are two cases: the file doesn't exist and the file exists but we got an error when opening it